e2e/tests/user/registration.spec.ts

```diff
@@ -61,7 +61,7 @@ describe('User registration tests:', () => {
     await createAccountPage.fillOutInstitution();
     nextButton = await createAccountPage.getNextButton();
     await nextButton.waitUntilEnabled();
-    await nextButton.click();
+    await nextButton.clickWithEval();
```

Comment:
occasionally, Puppeteer `click()` on the `Next` button is not working. Changing to call to `Document.click()` function.